native-image-workshop/README.md

```diff
@@ -1,51 +1,59 @@
 # GraalVM Native Image Workshop
 
+## Prerequisites
+
+* OS : Linux or Mac OSX
+* Apache Maven
+
 ## Introduction
 
-This workshop aims to walk you through a number of features GraalVM's Native Image offers,
-namely:
+We are going to use a fairly trivial application to walk through using GraalVM Native Image. Along the way we will see:
 
-1. Run a basic Java app
-2. Build a native image from it and see how that compares in terms of startup time
-3. Look at how we use the tracing agent to identify any dynamic features
-4. Run some of the application at the native image build time
+1. How you can turn a Java app into a native executable
+2. How you can build a GraalVM Native IMage that works with the dynamic parts of Jva
+3. Using the command line tools for generating a native image, as well as using the Maven tooling
 
-## Setup
+So what is going to be our testbed application? We will use a command line Java application that counts the number of files within the current directory and sub directories. As a nice extra it also calculates their total size.
 
-Install GraalVM. Instructions can be found [here](./SETUP-QUICK.md).
+As you work through this workshop you will need to update the code, in the following steps.
 
-You need a working version of Maven as well, and this should be covered in the installation instructions.
+This workshop relies on using GraalVM 21.3 or higher, with the GraalVM Native Image package installed. It will work with either Community Edition (CE), or Enterprise Edition (EE). If there is any EE speific code / configuration, this will be clearly marked out.
 
-## Outline of Demo
+Let's beign!
 
-We are going to use a fairly trivial application to walk through some of the features that
-are available with GraalVM's Native Image. These are:
+## Setup
 
-1. How you can turn a Java app into a native executable
-2. How you can deal with Java Reflection API calls, etc..
-3. Using the command line tools for generating a native image, as well as using the Maven tooling
+Install GraalVM. Instructions can be found [here](https://www.graalvm.org/docs/getting-started/.
```

Comment:
I would maybe narrow down the search.

Instructions can be found [on the website](https://www.graalvm.org/docs/getting-started/):
* [Linux](https://www.graalvm.org/docs/getting-started/linux/)
* [macOS](https://www.graalvm.org/docs/getting-started/macos/)
* [Windows](https://www.graalvm.org/docs/getting-started/windows/)

Reply:
Updated in-line with your comments